server/py/services/api/api/endpoints/alert_activations.py

```diff
@@ -0,0 +1,90 @@
+# Copyright 2024 Iguazio
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+
+from typing import Optional
+
+from fastapi import APIRouter, Depends, Query
+from sqlalchemy.orm import Session
+
+import mlrun.common.schemas
+
+import framework.utils.auth.verifier
+import framework.utils.clients.chief
+import framework.utils.singletons.project_member
+import services.api.crud
+from framework.api import deps
+
+router = APIRouter()
+
+
+@router.get("/projects/{project}/alert-activations")
+async def list_alert_activations(
+    project: str,
+    name: Optional[str] = None,
+    since: Optional[str] = None,
+    until: Optional[str] = None,
+    entity: Optional[str] = None,
+    severity: Optional[list[str]] = None,
+    entity_kind: Optional[str] = None,
+    event_kind: Optional[str] = None,
```

Comment:
Also, verify the entity_kind in the system test to ensure it is covered
```suggestion
    entity_kind: Optional[str] = Query(None, alias="entity-kind"),
    event_kind: Optional[str] = Query(None, alias="event-kind"),
```